Kong Studio Bundle Documentation (#1996)
Kong/docs.konghq.com: app/studio/1.0.x/index.md

title: Kong Studio
---

> Kong Studio's base Insomnia Designer enables spec-first development for all REST and GraphQL services. 

Reviewer: ```suggestion
Kong Studio is an Enterprise module based on Insomnia Designer, which enables spec-first development for all REST and GraphQL services. 
```
Not sure if "module" is the right term here, but I'd suggest calling it out in some way.

Author: Let's use `bundle`

> Leveraging Insomnia Designer, organizations can accelerate design and test workflows via automated 

Reviewer: ```suggestion
By leveraging Insomnia Designer, organizations can accelerate design and test workflows via automated 
```

testing, direct Git sync, and inspection of all response types. Teams of all sizes 
can use Kong Studio to increase development velocity, reduce deployment risk, 
and increase collaboration.

Kong Studio enhances Insomnia Designer through plugins which enable organizations 

> the ability to generate configuration for Kong Enterprise and sync API designs 

Reviewer: ```suggestion
with the ability to generate configuration for Kong Enterprise and sync API designs 
```
